wiki/redirect.yaml

```diff
@@ -465,8 +465,9 @@
 "tips_and_tricks_on_skill_improvement":                       "Guides/Tips_and_Tricks_on_Skill_Improvement"
 "videos_from_youTube":                                        "Guides/Videos_From_YouTube"
 
-"help_center":                 "Help_Center"
-"help":                        "Help_Center"
+"help_center":                 "Help_Centre"
+"help_centre":                 "Help_Centre"
```

Comment:
wait, did I ask you to put this one here?

Reply:
yes
> add the other varient too (so other links and American English people can go to the right location)

Reply:
I'm sure that was me talking about keeping `help_center` (not `help_centre`). Anyways, remove it, case sensitive redirects aren't needed since osu-web handles it better.